config/locales/metadata/en.yml

```diff
@@ -436,7 +436,7 @@ en:
 
         data_release_delay_reason:
           label: Reason for delaying release
-          help: "To apply for a delay, please send the following information to <%= configatron.data_sharing_contact.name %> (<%= configatron.data_sharing_contact.email %>):<ul><li>SAC sponsor</li><li>Study title and, where available, data set(s) ID</li><li>Study description (should describe the data types that will be produced)</li><li>Data sharing plan (to include the following):<ul><li>- Which repository/repositories will be used</li><li>- If data will be made available under a managed access mechanism</li><li>- When data will be shared</li></ul></li><li>Reason(s) for delaying data release</li></ul>"
+          help: "To apply for a delay, please contact <%= configatron.data_sharing_contact.email %>"
```

Comment:
Might it be worth wrapping the email address in an `<a href="mailto:address@here.com>` so that it becomes clickable to start the email?

Reply:
Thanks. Attached screenshot:  https://github.com/sanger/sequencescape/issues/3603#issuecomment-1468418670